ecommerce/frontend/src/components/pages/LoginPage.js

```diff
@@ -1,3 +1,107 @@
-export const LoginPage = () => {
-    return 'login'
-}
+import './LoginPage.less'
+import { Form, Input, Button, Row, Col } from 'antd'
+import { Link } from 'react-router-dom'
+import Column from 'antd/lib/table/Column'
+
+export const LoginPage = ({ onSubmit = () => { } }) => {
+    const [form] = Form.useForm()
+
+    const onFinish = values => {
+        console.log('Received values of form: ', values)
+        onSubmit(values)
+    }
+
+    const onFinishFailed = errors => {
+        console.log('Finish failed: ', errors)
+    }
+
+    const onLoginAsVendor = () => {
```

Comment:
Is the purpose of this function is to differentiate between user & vendor? Maybe you can define a state like `const [isVendor, setIsVendor] = setState(false)` set it true on click to the vendor submit button. Then return this value on onSubmit.

Reply:
I am not sure your solution would work @ozdenizdolu though, because doesn't that require first setting the state then clicking on a Login button? so like 2 steps instead of 1?